WordPress/Resources/en-AU.lproj/Localizable.strings

```diff
@@ -2687,31 +1571,9 @@ translators: Block name. %s: The localized block name */
 /* Username Placeholder */
 "Email or Username of the person that should receive your invitation." = "Email or Username of the person that should receive your invitation.";
 
-/* A placeholder for the username textfield.
-   Invite Username Placeholder */
-"Email or Username…" = "Email or Username…";
-
 /* Overlay message displayed when export content started */
 "Email sent!" = "Email sent!";
 
-/* No comment provided by engineer. */
-"Embed block previews are coming soon" = "Embed block previews are coming soon";
```

Comment:
I was quite surprised by all the deletion in so many of the localization files. I opened this one at random and checked for this string in the source one. It's there:

https://github.com/wordpress-mobile/WordPress-iOS/blob/db3c272c16535d8b8eba04a82c3ee4602ae01ee4/WordPress/Resources/en.lproj/Localizable.strings#L2754-L2755

I think our tooling, the download part at least, is not at fault here. On GlotPress the same locale shows a huge number of untranslated strings:

![image](https://user-images.githubusercontent.com/1218433/160067441-6cbbbf65-ab25-4876-b52a-0f3ab1daffb4.png)

Reply:
I took a deeper look at this today, because that was indeed worrying.

The good news is, those deletions are the right outcome and those strings (which have always been untranslated) should never have been in those `.strings` files in the first place, and new tooling actually fixed a bug here 🎉 

You can read the details of the investigation and our understanding of why it ended up that way in the thread of the Slack discussion you opened in #i18n: p1648204287401299/1648190527.375219-slack-C02AED43D 

The TL;DR is:
 - The old tooling [never used `?filters[status]=current` when downloading the export from GlotPress](https://github.com/wordpress-mobile/WordPress-iOS/blob/1406b807d3d4c037d35c3b46b6ae263e48db63d0/Scripts/update-translations.rb#L140) 😱 
 - This led to it including all the strings, including untranslated ones, and also including entries like `"somekey" = "";` for untranslated strings… which is why we had [that ugly `fix-translation` patching script in the first place back then](https://github.com/wordpress-mobile/WordPress-iOS/blob/1406b807d3d4c037d35c3b46b6ae263e48db63d0/Scripts/fix-translation), which replaced those empty entries with `"somekey" = "somekey"` (†)
 - The new tooling [correctly uses `filters[status]=current`](https://github.com/wordpress-mobile/release-toolkit/blob/dfca603ad4176fdc141c41ca2d309da362fb0f77/lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_l10n_helper.rb#L145) (which is [the default for the filter parameter of this new download action](https://github.com/wordpress-mobile/release-toolkit/tree/trunk/lib/fastlane/plugin/wpmreleasetoolkit/actions/ios#L81-L86)), which is why we no longer have those `"somekey" = ""` anymore — and even better, why we don't include any untranslated strings from the export to begin with 😄 

So in the end, the new tooling actually fixed an issue we barely knew we had from the old tooling 🎉 🍾  and all those lines which got deleted in the (non-Mag16) locales are expected and actually the correct state that we should have had all along, since those strings have never been translated in GlotPress ever!

---

_(†) When I worked on the new tooling and saw that I did not get `"somekey" = "";` entries anymore when downloading the exports from GlotPress, I remember back then thinking that this was probably because that `fix-translation` script was made to fix a legacy but in GlotPress including those entries… bug that would have since been fixed in GlotPress, would was my explanation then for why we did not get those `""` translations anymore and why that patch was not necessary anymore in the new tooling. Turns out that was NOT the reason, and the core explanation was because we failed to use `filters[status]=current` to begin with!_

Reply:
The even better good news, as @yuliyan pointed out in the thread, is that the new tooling now removing those useless untranslated entries in `.strings` files… will also reduce the size of the `.ipa`! 🎉 

_(I'd be curious by how much, btw. Too bad our metrics project [ref: paaHJt-37V-p2] will not be ready in time to do the before/after on that one 😛 but I guess we can take a look manually, or using [the new action I created a while ago to fetch that data from ASC](https://github.com/wordpress-mobile/release-toolkit/pull/173) after publishing 19.5 to compare it with 19.4 😃 )_